### PR TITLE
DEVDOCS-4445: [add] Orders v2, Add Brand to Orders Object

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4445]

## What changed?
* Add `brand` to Orders V2 schemas
* Update examples to include `brand`

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4445]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4445?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -2049,6 +2050,7 @@ components:
                   bin_picking_number: BIN
                   external_id: null
                   fulfillment_source: ''
+                  Brand: BigCommerce
```
Comment (bc-traciporter) [inline reference/orders.v2.oas2.yml:2053]:
```suggestion
                  brand: BigCommerce
```
Can you confirm if brand is with a lowercase b or uppercase B? I assume it is lowercase (brand).

File: reference/orders.v2.oas2.yml [hunk 2]
```diff
@@ -2177,6 +2180,7 @@ components:
                   bin_picking_number: ''
                   external_id: null
                   fulfillment_source: ''
+                  Brand: BigCommerce
```
Comment (bc-traciporter) [inline reference/orders.v2.oas2.yml:2183]:
```suggestion
                  brand: BigCommerce
```

File: reference/orders.v2.oas2.yml [hunk 3]
```diff
@@ -2232,6 +2236,7 @@ components:
                   bin_picking_number: ''
                   external_id: {}
                   fulfillment_source: ''
+                  Brand: BigCommerce
```
Comment (bc-traciporter) [inline reference/orders.v2.oas2.yml:2239]:
```suggestion
                  brand: BigCommerce
```

**Review by bc-traciporter — COMMENTED** [review]
Left comments

**Review by bc-traciporter — DISMISSED** [review]
LGTM

**Review by stanzikratelbc — APPROVED** [review]
LGTM Ready to merge

**Review by stanzikratelbc — DISMISSED** [review]
LGTM. Ready to merge.

**Comment by slsriehl** [comment]
fixed a mistake of mine. PR now reflects previous work rebased out by mistake.

**Review by slsriehl — APPROVED** [review]
approved pending successful lint.

**Pull request merged**